arch/arm/mach-imx/fiohab.c

```diff
@@ -97,7 +97,7 @@ static int fiovb_provisioned(void)
 #define SECURE_FUSE_BANK	(0)
 #define SECURE_FUSE_WORD	(6)
 #define SECURE_FUSE_VALUE	(0x00000002)
-#elif CONFIG_IMX8MM
+#elif CONFIG_IMX8M
```

Comment:
Are all using the same fuses?

Reply:
This was a change from @Tim-Anderson and I think it's correct.  But I can double check.

Reply:
These values are correct for 8MM, 8MP (per SRM: 2021-04) and 8MQ (per RM/SRM: 2020-04)